utils/s3/client.cc

```diff
@@ -931,7 +930,18 @@ class client::do_upload_file {
         });
         // If this request fails, complete_multipart_upload() throws, the upload should then
         // be aborted in .close() method
-        co_await _client->make_request(std::move(req));
+        co_await _client->make_request(std::move(req), [](const http::reply& rep, input_stream<char>&& in_) -> future<> {
```

Comment:
There's also `client::upload_sink_base::finalize_upload()` that needs the same handling

Reply:
Done